src/Symfony/Component/Process/Exception/ProcessRuntimeException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Exception;
+
+use Symfony\Component\Process\Process;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class ProcessRuntimeException extends RuntimeException
```

Comment:
does this intermediate class make sense in terms of exception hierarchy?
I'm not sure it does - let's use a trait instead ? (it should be `@internal`)

Reply:
or even nothing at all, as the only shared code is the `getProcess` method, which has no business logic. Importing the trait would be almost as big as writing the getter.

Reply:
I did this to avoid code duplicate. But right, I'll revert this part.